datafusion-cli/Cargo.toml

```diff
@@ -45,7 +45,6 @@ datafusion = { path = "../datafusion/core", version = "37.0.0", features = [
     "unicode_expressions",
     "compression",
 ] }
-datafusion-common = { path = "../datafusion/common" }
```

Comment:
I had to change this to
```toml
datafusion-common = { path = "../datafusion/common", version="37.0.0" }
```

to get `datafusion-cli` to publish. I think removing the whole dependency is cleaner that adding a version that needs to be kept in sync